src/app/appointments/AppointmentsList.tsx

```diff
@@ -0,0 +1,131 @@
+"use client";
+import { useRouter } from "next/navigation";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+import { Box, Tooltip } from "@mui/material";
+import Container from "@mui/material/Container";
+import Sheet from "@mui/joy/Sheet";
+import EditCalendar from "@mui/icons-material/EditCalendar";
+import Delete from "@mui/icons-material/Delete"; // Uncomment if you have a delete icon
```

Comment:
according to whom? 🤔 

```suggestion
import Delete from "@mui/icons-material/Delete";
```